distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numeric/Rounding_Mode.enso

```diff
@@ -0,0 +1,45 @@
+polyglot java import java.math.RoundingMode
+
+## A wrapper around the Java
+   [`RoundingMode`](https://docs.oracle.com/javase/8/docs/api/java/math/RoundingMode.html)
+    class.
+
+   `Rounding_Mode` is used to specify the method to use for rounding a value in
+   operations on `Decimal`s.` See `Decimal` for more details.
+type Rounding_Mode
```

Comment:
As discussed I think we can be opinionated here and just do default and bankers (not sure we even need that!)

Reply:
Note that the default in Java (half up) is not the default in Enso (asymmetric half-up). Java doesn't even offer asymmetric half-up, so we would have to add that ourselves.

Reply:
Done.

Reply:
> Note that the default in Java (half up) is not the default in Enso (asymmetric half-up). 

This is concerning.

> Java doesn't even offer asymmetric half-up, so we would have to add that ourselves.

Ah I see.

I think we should either:
1. Implement one ourselves (but that's definitely not for this PR and probably not really worth at all at this point).
2. Make it very clear for the user of this difference.

So I guess for now we'd go for (2). But indeed I think this has to be noted in a way that the user can find.